code/game/mecha/mecha.dm

```diff
@@ -501,6 +501,8 @@
 
 /obj/mecha/attack_animal(mob/living/simple_animal/user as mob)
 	log_message("Attack by simple animal. Attacker - [user].",1)
+	user.changeNext_move(CLICK_CD_MELEE)
+	user.do_attack_animation(src)

```

Comment:
https://github.com/ParadiseSS13/Paradise/blob/master/code/_onclick/other_mobs.dm#L46-L47
This is what calls `attack_animal` in most cases afaik